src/common/fetch/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const METHODS = {
```

Comment:
maybe it is better to use Enum instead of const?

Reply:
Tried to use enums, but this method parameter  is forwarded to axios it have to be string:(